ivy/functional/frontends/paddle/nn/functional/activation.py

```diff
@@ -193,6 +193,18 @@ def relu_(x, name=None):
     return x
 
 
+@with_supported_dtypes({"2.4.2 and below": ("float32", "float64")}, "paddle")
```

Comment:
We now use the paddle latest version `2.5.0`

Reply:
done , changed the paddle version to 2.5.0